src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -274,6 +273,13 @@ export class SearchSource {
    */
   fetch$(options: ISearchOptions = {}) {
     const { getConfig } = this.dependencies;
+    const syncSearchByDefault = getConfig(UI_SETTINGS.COURIER_BATCH_SEARCHES);
```

Comment:
This is the main change in this PR - replacing using `msearch` with setting the strategy to `es` and relying on `bfetch` to do the batching.